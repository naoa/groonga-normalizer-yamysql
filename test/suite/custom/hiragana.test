register normalizers/yamysql

normalize NormalizerYaMySQL "ぁあぃいぅうぇえぉおかがきぎくぐけげこごさざしじすずせぜそぞただちぢっつづてでとどなにぬねのはばぱひびぴふぶぷへべぺほぼぽまみむめもゃやゅゆょよらりるれろゎわゐゑをん"

