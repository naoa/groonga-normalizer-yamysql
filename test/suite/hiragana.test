register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "ぁあぃいぅうぇえぉおかがきぎくぐけげこごさざしじすずせぜそぞただちぢっつづてでとどなにぬねのはばぱひびぴふぶぷへべぺほぼぽまみむめもゃやゅゆょよらりるれろゎわゐゑをん"

