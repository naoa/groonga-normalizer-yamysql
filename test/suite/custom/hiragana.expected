register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQL "ぁあぃいぅうぇえぉおかがきぎくぐけげこごさざしじすずせぜそぞただちぢっつづてでとどなにぬねのはばぱひびぴふぶぷへべぺほぼぽまみむめもゃやゅゆょよらりるれろゎわゐゑをん"
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ぁあぃいうぇええぉおかがきぎくぐけげこごさざしじすずせぜそぞただちじっつずてでとどなにぬねのはばぱひびぴふぶぷへべぺほぼぽまみむめもゃやゅゆょよらりるれろゎわいえをん",
    "types": [

    ],
    "checks": [

    ]
  }
]
